Modules/IO/JPEG/src/itkJPEGImageIO.cxx

```diff
@@ -205,6 +201,10 @@ JPEGImageIO::Read(void * buffer)
   // read the header
   jpeg_read_header(&cinfo, TRUE);
 
+  // jpeg_calc_output_dimensions used in ReadImageInformation,
+  // so has to used here too
+  jpeg_calc_output_dimensions(&cinfo);
+
```

Comment:
_jpeg_calc_output_dimensions_ is used in _ReadImageInformation_ before, have to be used here too to ensure same parameters.